assignments/prototypes.js

```diff
@@ -39,6 +39,33 @@
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
 
+function GameObject(gameObejectAttributes){
+  this.createdAt = gameObejectAttributes.createdAt;
+  this.dimensions = gameObejectAttributes.dimensions;
+}
+
+GameObject.prototype.destroy = function () {
+  return `${this.name} was removed from the game`; 
+}
+
+function CharacterStats(characterStatsAttribute){
+  this.hp = characterStatsAttribute.hp;
```

Comment:
```suggestion
  this.healthPoints = characterStatsAttribute.healthPoints;
```
keep it as healthPoints for the console.logs below to work